contracts/Bridge.sol

```diff
@@ -60,13 +61,13 @@ contract Bridge is Pausable, AccessControl {
 
     event Retry(string txHash);
 
-    modifier onlyAdmin() {
-        _onlyAdmin();
+    modifier onlyAllowed(string memory func) {
```

Comment:
Could be bytes32 instead of a string to make things cheaper with the same semantics.
If we are just using the method names, then we could as well use msg.sig as ds-auth does.